### DISCUSSION TITLE
My BootStrapper Exe is not running when using Custom Theme

### DISCUSSION THREAD

**Discussion by amitcapg** [wixtoolset/issues#8823] [Questions]
I am using CustomTheme.xml to align my Exe Title and logo horizontally. Below is my CustomTheme.xml

<?xml version="1.0" encoding="UTF-8"?>
<Theme xmlns="http://wixtoolset.org/schemas/thmutil/2010">
   
   <Window Width="485" Height="300" HexStyle="100a0000" FontId="0">Welcome to My Application</Window>

   
   <Font Id="0" Family="Segoe UI" Size="9" Bold="yes" />
   <Font Id="1" Family="Segoe UI" Size="12" Bold="yes" />

   
   <Page Name="Main">
     
     <Image Name="LogoImage" X="11" Y="11" Width="50" Height="50" File="Logo.png" />

     
     <Text X="70" Y="20" Width="400" Height="30" FontId="1" DisablePrefix="yes">Welcome to the Installer</Text>
   </Page>
</Theme>

Below is how I am referring to it in my Bundle.wxs

<BootstrapperApplicationRef Id="WixStandardBootstrapperApplication.RtfLicense" >
  <bal:WixStandardBootstrapperApplication
    ShowVersion="yes"
    LicenseFile="EULA.rtf" ThemeFile="CustomTheme.xml" />
</BootstrapperApplicationRef>

Logo.png, EULA.rtf and CustomTheme.xml are saved at the same location where Bundle.wxs is saved. 
BootStraper Exe is not running when I use CustomTheme.xml in ThemeFile and runs when I don't use it (only use EULA.rtf). Could anyone please tell me what the error I am doing is?

Below is the error I am getting

[4490:0E20][2024-11-08T13:38:12]e000: Error 0x8007000d: Failed to load theme from path: C:\Users\asriva10\AppData\Local\Temp\{3061C450-5F1B-42E2-9337-4D37E12F9088}\.ba\thm.xml
[4490:0E20][2024-11-08T13:38:12]e000: Error 0x8007000d: Failed to initialize data in bootstrapper application.

**Comment by wmanning** [comment]
Not enough information.  What does your ```<BootstrapperApplication>``` element look like?
https://wixtoolset.org/docs/schema/wxs/bootstrapperapplication/
